### PR TITLE
FeedTAXII2: return report object with the relationships

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-25648

## Description
Return the report object with it's relationships.

### CODE REVIEW

File: Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py [hunk 1]
```diff
@@ -410,23 +410,37 @@ def get_indicator_publication(indicator: Dict[str, Any]):
...
+        obj_refs = report_obj.get('object_refs', [])
+        relationships: list[EntityRelationship] = []
+        for related_obj in obj_refs:
+            relation_type_and_id = related_obj.split('--')
+            relationships.append(
+                EntityRelationship(
+                    name='related-to',
+                    entity_a=report_obj.get('name'),
+                    entity_a_type='Report',
+                    entity_b=relation_type_and_id[1],
```
Comment (Ni-Knight) [inline Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py:436]: We can't relate to a hash, its an invalid value for all types. We have to search the actual STIX envelope for the object.

**Review by Ni-Knight — APPROVED** [review]
Looks good, just need to check it live

File: Packs/FeedTAXII/ReleaseNotes/1_1_28.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### TAXII 2 Feed
+
+- Fixed an issue where report objects not parsed correctly.
```
Comment (DeanArbel) [inline Packs/FeedTAXII/ReleaseNotes/1_1_28.md:6]:
```suggestion
- Fixed an issue where report objects were not parsed correctly.
```

**Review by DeanArbel — APPROVED** [review]
Well done

**Pull request merged**